frame/ddc-metrics-offchain-worker/src/lib.rs

```diff
@@ -66,15 +56,18 @@ struct MetricInfo {
     appPubKey: String,
     #[serde(deserialize_with = "de_string_to_bytes")]
     partitionId: Vec<u8>,
-    bytes: u128,
-    requests: u128,
+    storageBytes: u128,
+    wcuUsed: u128,
+    rcuUsed: u128,
 }
 
+#[allow(non_snake_case)]
 #[derive(Default, Debug)]
 struct DDNMetricInfo {
-	ddn_id: Vec<u8>,
-	bytes: u128,
-	requests: u128,
+    p2p_id: String,
+    storageBytes: u128,
```

Comment:
The better name would be `bytes_stored`

Reply:
Maybe, probably not worth going back to DDN and SC to change it though.